server/index.js

```diff
@@ -58,12 +58,39 @@ app.post(routes.API.TELEMMETRY, function (req, res) {
  * @param  {} routes.API.USER_SEARCH This is the user search api url
  * @param  {} res
  * @param  {} {res.status(200) Sending the api response
- * @param  {[]}} .json({users}) Sending empty array response 
+ * @param  {[]}} .json({users}) Sending empty array response
  */
 app.post(routes.API.USER_SEARCH, function (req, res) {
   res.status(200).json({ users: [] })
 });
 
+/**
+ * @param  {} routes.API.USER_ROLE This is the user role api url
+ * @param  {} res
+ * @param  {} {res.status(200) Sending the api response
+ * @param  {[]}} .json({userRoles}) Sending array response
+ */
+ app.get(routes.API.USER_ROLE, function (req, res) {
+  res.status(200).json(envVariables.USER_ROLE)
+});
+
+/**
+ * @param  {} routes.API.USERS This is the user role api url
+ * @param  {} res
+ * @param  {} {res.status(200) Sending the api response
+ * @param  {[]}} .json({userRoles}) Sending array response
+ */
+ app.post(routes.API.USERS, function (req, res) {
```

Comment:
Did you test with the environment variable?
what happens if the user role is not sent in the request body and what about validation of the request body for this API?

Reply:
If the roleType not sent in request body, empty users response will return.

Reply:
It should be standard way to validate as API should return proper status codes and handle other scenarios